### PR TITLE
Huge expansion to defaultentities.js

### PR DESCRIPTION
**Pull request by Danielv123**
^ explains everything. Look at commits for a bit more details.

### CODE REVIEW

File: defaultentities.js [hunk 1]
```diff
@@ -46,6 +46,154 @@ module.exports = { // ADD MORE (vanilla) AS YOU PLEASE (or modded if it's just f
...
+    type:item,
+    width:1,
+    height:1
+  },
+  straight_rail: {
+    type:item,
+    width:2,
+    height:2
+  },
+  // how do I handle curved rails?`
```
Comment (demipixel) [inline defaultentities.js]: Could you add curved rails with a with/height of 1 and just add a comment saying we'll figure this out later?

File: defaultentities.js [hunk 2]
```diff
@@ -46,6 +46,154 @@ module.exports = { // ADD MORE (vanilla) AS YOU PLEASE (or modded if it's just f
...
+    type: item,
+    width: 1,
+    height: 1
+  },
+  express_underground_belt: {
+    type: item,
+    width: 1,
+    height: 1
+  },
+  basic_splitter: { // how do we deal with rotatable items?
```
Comment (demipixel) [inline defaultentities.js]: They are already handled, you just need the default width and height when you bring it out.

**Comment by demipixel** [comment]
Looks good, added some comments to the changes. Could you fix those? I'll be happy to merge afterwards :)

**Comment by Danielv123** [comment]
Issues fixed, are we ready for a merge now?

**Pull request merged**